inc/pagination.php

```diff
@@ -42,12 +42,21 @@ function hale_archive_pagination($template = '', $custom_query = '')
                     <?php
                         if ($current_page_number > "1") {
                             echo "<li class='archive-pagination-prev-btn'>";
-                            previous_posts_link('< Previous', $max_pages);
+                            previous_posts_link('
+                            <svg class="govuk-pagination__icon govuk-pagination__icon--prev" xmlns="http://www.w3.org/2000/svg" height="13" width="15" aria-hidden="true" focusable="false" viewBox="0 0 15 13">
+                                <path d="m6.5938-0.0078125-6.7266 6.7266 6.7441 6.4062 1.377-1.449-4.1856-3.9768h12.896v-2h-12.984l4.2931-4.293-1.414-1.414z"></path>
+                            </svg><span class="govuk-pagination__link-title">Previous<span class="govuk-visually-hidden"> page</span>
```

Comment:
Does the previous text need to be translated to welsh?

Reply:
Added a translation tag around the text.  
Note that there is a visually hidden suffix which will also need to be translated:
`Previous<span class="govuk-visually-hidden"> page</span>`
I have wrapped all of this in the translation tag - this is because a language might put the "page" bit before the "previous" bit.  It does mean that there is HTML in the translation, but I don't really see a big issue with that.